### PR TITLE
Convert statparser.php to python

### PR DESCRIPTION
**Pull request by druzsan**

### CODE REVIEW

File: yrstats.py [hunk 1]
```diff
@@ -160,7 +145,11 @@ def do(self, event):
 
         self.last_notification_time = time_now
 
-        gamestats = call_stat_dmp_parser(self.config, stat_dmp_file_path = event.src_path)
+        gamestats = statparser.process_stats(
```
Comment (angadsingh) [inline yrstats.py]: can you give an option to call the php based stat parser vs the python stat parser or are you confident that yours supports all the features/details extracted by the old one?

**Review by angadsingh — CHANGES_REQUESTED** [review]

**Comment by angadsingh** [comment]
thanks a tonne @druzsan. lets just give users an option between the two modes and keep it flexible till we know for sure that the python version runs as well as the php version.

**Comment by druzsan** [comment]
@angadsingh I'm quite sure that `python` statparser is confident with the `php` one, but it is not a problem for me to add the choise option.
One can add __`--use-php-parser`__ option to `extract-game-stats` and `start-stat-watcher` commands in order to run these with the `php` parser. By default they run with the `python` one.
I also have changed the `Lost` suffix in detailed counts to `Left` in both parsers.

File: yrstats.py (hunk 1 above)
Comment (druzsan) [reply yrstats.py]: done

**Comment by angadsingh** [comment]
Thanks @druzsan. Power to you.

**Pull request merged**